service-worker.js

```diff
@@ -0,0 +1,20 @@
+// This is the service worker with the Cache-first network
+
+const CACHE = "pwabuilder-precache";
+
+importScripts(
```

Comment:
@aoliveiramagalhaes this should give offline support for the app once installed 🙏 

but it needs to be tested!

* install the app
* turn airplane mode one
* try to open and play
* 🙏